template-roles-and-resources.md

```diff
@@ -0,0 +1,167 @@
+- Strive to give feedback that is constructive and avoid making comments that will increase defensiveness.
+- Be available (asynchronously) to the templateer if they have follow-up questions or concerns about your feedback.
+
+
+### Eligibility
+
+To become a community reviewer:
+- You should be a member in good standing in the Good Docs Project community, as defined in our [Code of Conduct](https://thegooddocsproject.dev/code-of-conduct/).
+- You should have writing experience or subject matter expertise for the type of document being reviewed.
+- You should be familiar with the [Conventional Comments](https://conventionalcomments.org/) theory of labeling feedback.
+- Let the current templates coordinator or a template mentor know that you want to volunteer as a community reviewer.
+
+
+## Pull request reviewers
+
+This group of experienced templateers review and approve pull requests to the templates repository.
+
+> :triangular_flag_on_post: **NOTE: At least two pull request reviewer approvals are necessary to merge a template into the project.**
+
+The current pull request reviewers are:
+
+- Aidan Doherty
+- Alyssa Rock
+- Cameron Shorter
+
+### Responsibilities
+
+Pull request reviewers:
+- Act as gatekeepers for template quality and assist in making the final judgment call about when a template is ready to merge.
+- Apply the [template quality checklist](template-quality-checklist.md) to each pull request to ensure it meets the project’s standards for quality.
+- Strive to give feedback that is constructive and avoid making comments that will increase defensiveness.
+- Be available (asynchronously) to the templateer if they have follow-up questions or concerns about your feedback.
+- Coordinate and communicate regularly with the templates coordinator.
+- Work with the templates coordinator to ensure each template receives at least two reviews from pull request reviewers when a template is in the pull request phase.
+- Notify the templates coordinator and make recommendations about when a template is ready to be merged.
+
+
+### Eligibility
+
+To become a pull request reviewer:
+- You should be a member in good standing in the Good Docs Project community, as defined in our [Code of Conduct](https://thegooddocsproject.dev/code-of-conduct/).
+- You should be familiar with the Conventional Comments theory of labeling feedback.
+- You should be an experienced technical writer who is familiar with our project’s standards for quality.
+- You should have experience with the template writing process, which is demonstrated by one or more of the following:
+  - Completing a full template set from beginning to end.
+  - Participating in a template writing working group for at least 2 months.
+- You must be nominated for this role by the existing templates coordinator or a member of the project steering committee.

```

Comment:
question: Should template mentors be able to nominate pull request reviewers?

Reply:
Yes, I think that's reasonable.

Reply:
I've adopted this suggestion.